CarpetXRegrid/src/regrid_error.cxx

```diff
@@ -1,18 +1,17 @@
-#include "schedule.hxx"
-
 #include <loop_device.hxx>
 
 #include <cctk.h>
 #include <cctk_Arguments.h>
 #include <cctk_Parameters.h>
 
 namespace CarpetX {
+using namespace std;
 
 extern "C" void CarpetX_InitError(CCTK_ARGUMENTS) {
   DECLARE_CCTK_ARGUMENTSX_CarpetX_InitError;
   DECLARE_CCTK_PARAMETERS;
 
-  grid.loop_device<1, 1, 1, where_t::everywhere>(
+  grid.loop_device<1, 1, 1, Loop::where_t::everywhere>(
```

Comment:
Maybe use the simpler `loop_all_device` instead?

Reply:
done